ramls/schemas/jobExecution.json

```diff
@@ -34,6 +34,10 @@
       "type": "string",
       "$ref": "../../ramls/raml-util/schemas/uuid.schema"
     },
+    "jobProfileName": {
```

Comment:
make this a read only field that way we can just populate it in the GET call for jobExecution

Reply:
That way the job profile name is never stale